pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>org.cognitor.cassandra</groupId>
     <artifactId>cassandra-migration-parent</artifactId>
-    <version>2.6.2_v4-SNAPSHOT</version>
+    <version>2.6.3_v4-SNAPSHOT</version>
```

Comment:
Can you just revert this change? I change the version number depending on the change that was done for the release. If the version number is change now the main package and the spring boot package have different versions.
Once that change is reverted I will merge the remaining changes.